juicy-html.html

```diff
@@ -22,7 +22,10 @@
             }
             constructor(self) {
                 self = super(self);
-                var model = null; // XXX(tomalec): || this.model ? // to consider for https://github.com/Juicy/juicy-html/issues/30
+                
+                // Idea: Handle pre-upgrade state
+                // https://github.com/Juicy/juicy-html/issues/30
+                var model = null || this.model;
```

Comment:
This is simpler, easier to read:

```suggestion
                var model = this.model;
```

Reply:
```suggestion
                var model = this.model || null;
```

This is what I wanted originally, to don't have `undefined` value. Thank you for spotting.